content/docs/releases/release-notes/release-notes-1.13.md

```diff
@@ -3,6 +3,70 @@ title: Release 1.13
 description: 'cert-manager release notes: cert-manager 1.13'
 ---
 
+## v1.13.3
+
+This patch release contains fixes for the following security vulnerabilities in the cert-manager-controller:
+- [`GO-2023-2334`](https://pkg.go.dev/vuln/GO-2023-2334): Decryption of malicious PBES2 JWE objects can consume unbounded system resources.
```

Comment:
FYI `govulncheck` shows that there are other vulnerabilities in go 1.20.10 which we haven't patched yet,
so I guess we'll have to do another patch release :-( soonish.

```sh
$ crane export quay.io/jetstack/cert-manager-controller:v1.13.2 - | tar -x -O app/cmd/controller/controller > /tmp/cert-manager-controller-v1.13.2 && govulncheck -mode=binary /tmp/cert-manager-controller-v1.13.2
Scanning your binary for known vulnerabilities...

Vulnerability #1: GO-2023-2382
    Denial of service via chunk extensions in net/http
  More info: https://pkg.go.dev/vuln/GO-2023-2382
  Standard library
    Found in: net/http/internal@go1.20.10
    Fixed in: net/http/internal@go1.21.5
    Example traces found:
      #1: internal.chunkedReader.Read

Vulnerability #2: GO-2023-2334
    Decryption of malicious PBES2 JWE objects can consume unbounded system
    resources
  More info: https://pkg.go.dev/vuln/GO-2023-2334
  Module: github.com/go-jose/go-jose/v3
    Found in: github.com/go-jose/go-jose/v3@v3.0.0
    Fixed in: github.com/go-jose/go-jose/v3@v3.0.1
    Example traces found:
      #1: jose.JSONWebEncryption.Decrypt
      #2: jose.JSONWebEncryption.DecryptMulti

Vulnerability #3: GO-2023-2186
    Incorrect detection of reserved device names on Windows in path/filepath
  More info: https://pkg.go.dev/vuln/GO-2023-2186
  Standard library
    Found in: path/filepath@go1.20.10
    Fixed in: path/filepath@go1.21.4
    Example traces found:
      #1: filepath.IsLocal

Vulnerability #4: GO-2022-0646
    Use of risky cryptographic algorithm in github.com/aws/aws-sdk-go
  More info: https://pkg.go.dev/vuln/GO-2022-0646
  Module: github.com/aws/aws-sdk-go
    Found in: github.com/aws/aws-sdk-go@v1.45.7
    Fixed in: N/A
    Example traces found:
      #1: s3crypto.NewDecryptionClient
      #2: s3crypto.NewEncryptionClient

Your code is affected by 4 vulnerabilities from 2 modules and the Go standard library.

```